static/app/views/issueDetails/groupDetails.tsx

```diff
@@ -273,6 +270,19 @@ function useEventApiQuery(
   return isLatest ? latestEventQuery : otherEventQuery;
 }
 
+function useSyncGroupStore() {
+  const queryClient = useQueryClient();
+  const router = useRouter();
+  const allGroups = useLegacyStore(GroupStore);
+
+  const params = router.params;
+  const group = allGroups.find(({id}) => id === params.groupId) as Group;
+
+  useEffect(() => {
+    setApiQueryData(queryClient, [`/issues/${params.groupId}/`], group);
```

Comment:
You probably also want to guard this with `if (defined(group))` since it could be undefined. You can also remove the dependency for `params.groupId` if you use `group.id` instead.